_podcasts/4-bitcoin-core-pre-21-disclosures.md

```diff
@@ -0,0 +1,35 @@
+---
+title: "Episode 4: Discussing Pre-0.21.0 Bitcoin Core Vulnerability Disclosures"
+permalink: /podcast/4-bitcoin-core-pre-21-disclosures/
+layout: podcast
+date: 2024-07-11
+description: Gloria Zhao and Niklas Gögge discuss the recently disclosed Bitcoin Core pre-0.21.0 vulnerabilities.
+---
+
+Brink engineers [Gloria Zhao][] and [Niklas Gögge][] talk through the recently
+disclosed Bitcoin Core pre-0.21.0 vulnerabilities.
+
+<iframe src="https://podcasters.spotify.com/pod/show/bitcoinbrink/embed/episodes/Sebastian-Falbesoner-on-Code-Review-and-BIP324-e2909ue" height="102px" width="400px" frameborder="0" scrolling="no"></iframe>
```

Comment:
I assume this will be replaced with the new one so it's not the Sebastian one, but leaving comment just in case